umbral/bignum.py

```diff
@@ -81,6 +81,34 @@ def from_int(cls, num, curve: ec.EllipticCurve=None):
 
         return cls(bignum, curve_nid, group, order)
 
+    @classmethod
+    def hash_to_bn(cls, *crypto_items, params=None):
+        params = params if params is not None else default_params()
+
+        # TODO: Clean this in an upcoming cleanup of pyUmbral
+        blake2b = hashes.Hash(hashes.BLAKE2b(64), backend=backend)
+        for item in crypto_items:
+            try:
+                item_bytes = item.to_bytes()
```

Comment:
During our discussion, we talked about only allowing bytes and nothing else.  I thought you were on board for that (although I can see the allure of taking arbitrary `crypto_items`).  Where are you standing at this point?